creator-node/src/utils.js

```diff
@@ -95,7 +95,11 @@ const ipfsSingleByteCat = (path, logContext, timeout = 1000) => {
       logger.info(`ipfsSingleByteCat - Retrieved ${path} in ${Date.now() - start}ms`)
       resolve()
     } catch (e) {
-      logger.error(`ipfsSingleByteCat - Error: ${e}`)
+      // Expected message for e is `TimeoutError: Request timed out`
+      // if it's not that message, log out the error
+      if (!e.message.includes('Request timed out')) {
```

Comment:
feels like we'd want to keep this log stmt? is useful to see, for example, how frequently these requests are timing out. maybe lower from error to info. wdyt?

given these numbers:
```
RehydrateIpfsQueue: count: 0
9,244,789

RehydrateIpfsQueue: Adding a rehydrateIpfsFromFsIfNecessary task to the queue!
4,194,961

RehydrateIpfsQueue: Successfully added a rehydrateIpfsFromFsIfNecessary task!
3,997,661

ipfsSingleByteCat - Error: TimeoutError: Request timed out
834,008
```


seems cutting 800k compared to the above 15mm is not that substantial?

Reply:
The single byte cat operation expects to see the timeout error. That's how we know if the file exists on the local node or not. My thinking was since IPFS rehydrate happens in the background and the pod and data are ephemeral, wouldn't make sense to log except for cases where the error is not what we expect. The only case I think we need this is to debug rehydrate, and if you think that's significant enough to keep, we can change to info.

Reply:
eh fair enough. lets remove it for now and can re-evaluate later